### PR TITLE
fix(quantic): fixed width of inputs for numericFacet

### PR DESCRIPTION
**Pull request by SimonMilord**
[SFINT-4712](https://coveord.atlassian.net/browse/SFINT-4712)

Fixed an issue brought up following [ previous PR](https://github.com/coveo/ui-kit/pull/2666)

The numeric Facets and the apply button were not taking the full width of the container in the Full search.

BEFORE:

image

AFTER:

Screen Shot 2023-01-23 at 6 18 11 PM

[SFINT-4712]: https://coveord.atlassian.net/browse/SFINT-4712?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by nathanlb** [comment]
Please update de PR title to be in the proper format ex: "fix(quantic): numeric facet alignment"

File: packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.css [hunk 1]
```diff
@@ -3,4 +3,9 @@
 .facet__search-form {
   display: flex;
   align-items: flex-start;
+  justify-content: space-between;
+}
+
+.facet__search-formItem {
```
Comment (nathanlb) [inline packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.css:9]: IIRC we don't use camel casing in our css. 
```suggestion
.facet__search-form-item {
```

**Comment by SimonMilord** [comment]
> Please update de PR title to be in the proper format ex: "fix(quantic): numeric facet alignment"

oops I used my old branch and forgot to update the title, my bad

**Comment by nathanlb** [comment]
> > Please update de PR title to be in the proper format ex: "fix(quantic): numeric facet alignment"
> 
> oops I used my old branch and forgot to update the title, my bad

No worries; just a reminder 😄

**Review by nathanlb — APPROVED** [review]
looks legit just the css class nitpick

File: packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.css (hunk 1 above)
Comment (SimonMilord) [reply packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.css]: good to know, will stick with kebabs in the future! 🍢🍢🍢🍢

Comment (nathanlb) [reply packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.css]: We loosely follow the [BEM naming convention](https://getbem.com/naming/). Or at least we try to... 🙄

**Pull request merged**